_includes/core/value_props.html

```diff
@@ -0,0 +1,63 @@
+<div id="vp-container">
+  <div id="initial-vps">
+    <div class="row value-props">
+      <div class="col-lg-4 text-center animated fadeInLeft">
+        <span class="fa fa-cubes fa-3x"></span>
+        <h2 class="value-prop-heading">Modular</h2>
+        <p>.NET Core is a set of runtime, library and compiler components. Microsoft uses these components in various configurations for device and cloud workloads. You can do the same for your app or service.</p>
+      </div><!-- /.col-lg-4 -->
+      <div class="col-lg-4 text-center  animated fadeIn">
+        <span class="fa fa-code fa-3x"></span>
+        <h2 class="value-prop-heading">Managed</h2>
+        <p>Managed runtimes make code easy to write and guarantee safe execution. .NET Core manages memory with a garbage collector, compiles your code with a JIT compiler or ahead of time with .NET Native. </p>
+      </div><!-- /.col-lg-4 -->
+      <div class="col-lg-4 text-center animated fadeInRight">
+        <span class="fa fa-linux fa-3x"></span>
+        <h2 class="value-prop-heading">Cross-platform</h2>
+        <p>You can create .NET Core apps that run on multiple OSes and CPUs. .NET Core runs on Windows. Ports are in progress for Linux, OS X and FreeBSD, as is integration with the LLVM compiler.</p>
+      </div><!-- /.col-lg-4 -->
+    </div><!-- /.row -->
+  </div>
+  <div id="rest-vps">
+    <div class="row value-props">
+      <div class="col-lg-4 text-center">
+        <span class="fa fa-magic fa-3x"></span>
+        <h2 class="value-prop-heading">Modern</h2>
+        <p>.NET Core brings with it a set of languages, led  by C# with support for modern language features, like generics, Language Integrated Query (LINQ), async support and more. It is backed by a managed compiler called "Roslyn" that is exposed to the runtime, usable as a service.</p>
```

Comment:
Is it worth highlighting the languages that you can write with against .NET Core. I get asked about VB and F# as well as C#